docker/federation/lemmy_epsilon.hjson

```diff
@@ -10,4 +10,8 @@
   database: {
     host: postgres_epsilon
   }
+  pictrs: {
+    api_key: "my-pictrs-key"
+    image_proxy: true
```

Comment:
I forget if you've added a test for one of the ones to make sure it *isn't* proxying images for one of the ones where `image_proxy: false` . But it would be good to add a test for that case too.

Reply:
Good point, added.